src/components/ContextMenu/ContextMenuWrapper.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import { Trans } from 'react-i18next'
+import styled from 'styled-components'
+import Box from 'components/base/Box'
+import { colors } from 'styles/theme'
+
+// $FlowFixMe
+export const ContextMenuContext = React.createContext({})
+
+export const withContextMenuContext = (ComponentToDecorate: React$ComponentType<*>) => (
```

Comment:
this is very cool! side note for later, you could experiment around `useContext` hehe